src/block/profile.css

```diff
@@ -1,145 +1,143 @@
+  cursor: pointer;
+}
+.profile__avatar-container:hover .profile__avatar-edit-button {
+  opacity: 1;
+}
+
+@media screen and (max-width: 320px) {
+  .profile {
+    flex-direction: column;
+    text-align: center;
   }
   .profile__info {
-    display: grid;
-    grid-template-areas: "title button" "description descrpition";
-    grid-template-columns: minmax(auto, 373px) auto;
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    justify-content: center;
+    padding-bottom: 36px;
   }
-  .profile__edit-button {
-    background: transparent url(../images/JC\ edit\ button.svg) center no-repeat;
-    height: 24px;
-    width: 24px;
-    grid-area: button;
-    border: 1px solid #fff;
-    margin-left: 18px;
-    margin-top: 15px;
+  .profile__description {
+    position: absolute;
+    bottom: 75px;
+    text-align: center;
+    font-size: 15px;
+    line-height: 17px;
   }
   .profile__add-button {
-    background: transparent url(../images/Addition\ sign.svg) center no-repeat;
-    width: 150px;
     height: 50px;
-    border: 2px solid #fff;
-    margin-left: auto;
-  }
-  .profile__add-button:hover {
-    cursor: pointer;
-    opacity: 0.6;
-  }
-  .profile__edit-button:hover {
-    cursor: pointer;
-    opacity: 0.6;
+    width: 277px;
   }
   .profile__title {
-    line-height: 48px;
-    font-weight: 500;
-    margin: 0px;
-    white-space: nowrap;
-    text-overflow: ellipsis;
-    overflow: hidden;
-    font-size: 42px;
-    max-width: 400px;
-    padding-bottom: 7px;
+    flex-direction: column;
+    justify-content: center;
+    display: flex;
+    text-align: center;
+    font-size: 22px;
+    padding-top: 20px;
+    margin-left: 39px;
   }
-  .profile__description {
-    font-size: 18px;
-    line-height: 22px;
-    font-weight: 400;
-    margin: 0;
-    grid-area: description;
-    max-width: 400px;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-    overflow: hidden;
+  .profile__edit-button {
+    width: 18px;
+    height: 18px;
+    padding-left: 10px;
+    padding-top: 4px;
+    margin-top: 35px;
   }
   .profile__image {
-    height: 120px;
-    width: 120px;
-    border-radius: 50%;
-    object-fit: cover;
-    background-position: center;
-    margin-right: 24px;
-  }
-  @media screen and (max-width: 697px) {
-    .profile {
-      flex-direction: column;
-      margin-bottom: 36px;
-      padding-left: 10px;
-    }
-    .profile__title {
-      max-width: 220px;
-    }
-    .profile__info {
-      display: flex;
-      flex-wrap: wrap;
-      width: 100%;
-      justify-content: center;
-      grid-template-columns: auto;
-    }
-    .profile__description {
-      width: 100%;
-      text-align: center;
-      max-width: 282px;
-    }
-    .profile__add-button {
-      margin: 0;
-      width: 100%;
-    }
+    justify-content: center;
+    padding-left: 82px;
+    padding-right: 80px;
+    margin-left: 101px;
+    margin-right: 99px;
```

Comment:
Please, make the **page** match the `mobile` design on 320px https://snipboard.io/bR15OE.jpg  . 


It's because of the `avatar` margins. 

And you have a lot of `left` paddings in the other elements, which shift the card section to the left